package.json

```diff
@@ -463,7 +463,13 @@
                 "description": "Various levels of logging shown in the debug console. When set to 'log' or 'verbose', the logs will also be written to a file."
               },
               "envFile": {
-                "type": "string",
+                "type": [
+                  "string",
+                  "array"
+                ],
+                "items": {
+                  "type": "string"
+                },
                 "description": "Absolute path to a file containing environment variable definitions.",
```

Comment:
The description should be updated to reflect the fact that both string and array are supported now.

Reply:
I've added a better description